app/src/main/res/values/strings.xml

```diff
@@ -1,234 +1,489 @@
+    <string name="language_options_reset">Reset Voice Search Language Settings</string>
+
+    <!-- This string is for the table column name (in the browser's 'Bookmarks' list) for the 'Name' of the bookmarked website. -->
+    <string name="col_name">Name</string>
+
+    <!-- This string is for the table column name (in the browser's 'Bookmarks' list) for the 'Tags' of the bookmarked website. -->
+    <string name="col_tags">Tags</string>
+
+    <!-- This string is for the table column name (in the browser's 'Bookmarks' list) for the 'Location' (or 'URL' or 'Address') of the bookmarked website. -->
+    <string name="col_url">Location</string>
+
+    <!-- This string is for the table column name (in the browser's 'Bookmarks' list) for the date of when the bookmarked website was most recently visited. -->
+    <string name="col_most_recent_visit">Most Recent Visit</string>
+
+    <!-- This string is for the table column name (in the browser's 'Bookmarks' list) for the date of when the website was bookmarked. -->
+    <string name="col_date_added">Added</string>
+
+    <!-- This string is for the table column name (in the browser's 'Bookmarks' list) for the date of when the bookmarked website was last edited. -->
+    <string name="col_date_last_modified">Last Modified</string>
+
+    <!-- This string is for the tooltip that appears upon hovering the 'Back' button in the browser's navigation bar.
+         'Back' refers to going back one page in 'History'. -->
+    <string name="back_tooltip">Back</string>
+
+    <!-- This string is for the tooltip that appears upon hovering the 'Forward' button in the browser's navigation bar.
+         'Forward' refers to going back one page in 'History'. -->
+    <string name="forward_tooltip">Forward</string>
+
+    <!-- This string is for the tooltip that appears upon hovering the 'Forward' button in the browser's navigation bar.
+         'Refresh' refers to reloading the current page. -->
+    <string name="refresh_tooltip">Refresh</string>
+
+    <!-- This string is for the tooltip that appears upon hovering the 'Home' button in the browser's navigation bar.
+         'Home' refers to the browser's start page (e.g., the Firefox Reality Home Page). -->
+    <string name="home_tooltip">Home</string>
+
+    <!-- This string is for the tooltip that appears upon hovering the 'Voice Search' (microphone) button in the
+         browser's navigation bar. 'Voice Search' refers to the browser's system for searching the Web using
+         speech-to-text recognition. -->
+    <string name="voice_search_tooltip">Voice Search</string>
+
+    <!-- This string is for the tooltip that when clicking the 'Info' (ⓘ) icon visible next to the current website's
+         URL in the URL bar. The button it labels, when pressed, shows information about the site. -->
+    <string name="site_info_tooltip">Show site info.</string>
```

Comment:
This is the only one of these tooltips that has a period at the end of the string so want to double-check that's expected

Reply:
in Firefox desktop it appears as `Show site information`, which I abbreviated as `Show site info.`

for context: the [tooltips for the browser-level chrome UI in VR](https://github.com/MozillaReality/FirefoxReality/files/2292046/UIS-40.Tooltips_001.pdf) appear on hover. they're not implemented yet, but the STR will be added to the Wiki page (currently: the STR are TBD).

if you have ideas on this, I'd love to hear:

- should we not abbreviate `information` as `info.`?
- should we stay consistent with Firefox desktop's usage (i.e., `Show site information`)?

Reply:
I think you can abbreviate it as "info.", especially if the space is more limited than on Firefox (to avoid concatenation). I didn't realize that was because it was abbreviated :)
On the other hand, if size constraint is not an issue (note localizations are often much longer than English), then I'd just stay consistent with Firefox then. Just my 2 cents :)

Reply:
Per  your conversation just left it as <b>Show site info.</b>